x-pack/legacy/plugins/ml/public/data_visualizer/components/field_data_card/examples_list/examples_list.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText, EuiToolTip } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+interface Props {
+  examples: any[];
+}
+
+function getExamples(examples: any[]) {
```

Comment:
This could be a React Component by itself.

Reply:
Created a separate `Example` component in https://github.com/elastic/kibana/pull/42685/commits/992c67e94e727b7b5b5f91cbc73eb954c6e13a0f